samples/nodes/src/main/java/com/gluonhq/emoji/samples/Demo.java

```diff
@@ -41,35 +44,40 @@
 import javafx.stage.Stage;
 
 import java.util.List;
+import java.util.function.Consumer;
 import java.util.stream.Collectors;
 
 public class Demo extends Application {
 
     @Override
     public void start(Stage primaryStage) {
 
-        // Get a single text string with all unicode characters for all emojis
-        String text = getAllEmojisAsUnicodeString();
-
-        // Convert text to Text (if delimiter is not empty) and Image nodes (all emojis)
-        List<Node> emojiNodes = TextUtils.convertToTextAndImageNodes(text);
-        emojiNodes.stream()
-                .filter(ImageView.class::isInstance)
-                .forEach(node -> {
-                    String unified = (String) node.getProperties().get(EmojiImageUtils.IMAGE_VIEW_EMOJI_PROPERTY);
-                    EmojiData.emojiFromCodepoints(unified).ifPresent(emoji ->
-                            Tooltip.install(node, new Tooltip("Emoji: " + emoji.getName() + "\n" + unified)));
-                });
-        // Add nodes to textFlow
         TextFlow textFlow = new TextFlow();
-        textFlow.getChildren().addAll(emojiNodes);
         ScrollPane pane = new ScrollPane(textFlow);
         Scene scene = new Scene(pane, 1230, 800);
         textFlow.prefWidthProperty().bind(scene.widthProperty().subtract(30));
         primaryStage.setScene(scene);
         primaryStage.setTitle("Emoji nodes: " +
                 textFlow.getChildren().stream().filter(ImageView.class::isInstance).count());
         primaryStage.show();
+
+        EmojiLoaderFactory.getEmojiImageLoader().initialize().thenAccept(aBoolean -> {
```

Comment:
This demo could use a progress bar/indicator for the first load of the emoji sprites.

Reply:
Done!